source/use-cases/sensitive-data-encryption.txt

```diff
@@ -13,26 +13,51 @@ Sensitive Data Encryption
 Introduction
 ------------
 
-Cras sed rutrum nisi, vitae tristique risus. Aliquam erat volutpat.
-Donec auctor nisl eget ornare accumsan. Pellentesque lectus tortor,
-consectetur molestie pellentesque ac, blandit non libero.
+If you store information in MongoDB such as confidential personal details, payment information, or proprietary data, you are probably interested in protecting your data from unauthorized access. In addition, the data you store may be subject to governance, privacy, and security compliance mandates. 
+
+MongoDB offers several methods that reduce your risk of a data breach resulting from unauthorized access whether accidental or by a malicious act including:
+
+* `Role-based access control <https://docs.mongodb.com/manual/core/authorization/>`_
+* `Monitoring and logging <https://docs.mongodb.com/manual/core/auditing/>`_
+* `TLS/SSL network transport encryption <https://docs.mongodb.com/manual/core/security-transport-encryption/>`_
+* `Data encryption at rest <https://docs.mongodb.com/manual/core/security-encryption-at-rest/>`_
+* `Client-side field level encryption <https://docs.mongodb.com/manual/core/security-client-side-encryption/>`_
+
+This document guides you through setup and implementation of Client-Side Field Level Encryption (CSFLE), a feature that encrypts document fields specified by a developer, in a example scenario to demonstrate how it can be used to secure your data.
+
+.. note::
+
+  Client-side Field Level Encryption is available starting in MongoDB 4.2 Enterprise only
```

Comment:
To reduce repetition, I could probably reference the Requirements section here instead: "Check the requirements section to see if CSFLE is available for your driver and server." Let me know if that makes more sense.

Reply:
I think this note is good as it is for now. Once we have the page built out a bit more fully we can revisit.